packages/client/screens/reset-password/reset-password-sates/resetPasswordSuccessScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { View } from "../../../components/Themed";
+import { Button, Link, Text, VStack } from "native-base";
+import { StyleSheet } from "react-native";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { farEnvelope } from "../../../components/icons/regular/farEnvelope";
+import { openInbox } from "react-native-email-link";
+
+export default function ResetPassworSuccessScreen() {
+    return (
+        <View style={styles.container}>
+            <VStack style={styles.content}>
+                <FontAwesomeIcon icon={farEnvelope} color={"#0077E6"} style={styles.envelopeIcon} size={100} />
```

Comment:
As previously mentioned, this should be a light font weight envelope icon with size 96
```suggestion
                <FontAwesomeIcon icon={farEnvelope} color={"#0077E6"} style={styles.envelopeIcon} size={96} />
```